modules/olm-installing-opm.adoc

```diff
@@ -63,7 +63,7 @@ $ oc image extract {registry-image} \
 ifndef::openshift-origin[]
     -a ${REG_CREDS} \
 endif::[]
-    --path /usr/bin/registry/darwin-amd64-opm:. \
```

Comment:
@skrthomas So it looks like you removed the `registry/` path which is good, but you also changed the name of the binary from `darwin-amd64-opm` to just `opm`.

I'm not sure if that change is appropriate or not, but that's something that I would ask @zhouying7780 to confirm.

Or maybe @adellape knows a good SME on the Operator team to ask?

Reply:
Just double-checked, and they should remain as `darwin-amd64-opm` and `windows-amd64-opm` for those binaries.

Reply:
```
$ oc image extract registry.redhat.io/openshift4/ose-operator-registry:v4.6     -a ${REG_CREDS}     --path /usr/bin/windows-amd64-opm:. --confirm

W0128 10:31:30.601994  176229 manifest.go:414] Chose linux/amd64 manifest from the list.
To include the manifest list digest: sha256:490035a0394c1f6aa32f193b61a28f983801ef467339729d4a724bb4b3ec7f7e, use --keep-manifest-list=true

$ oc image extract registry.redhat.io/openshift4/ose-operator-registry:v4.6     -a ${REG_CREDS}     --path /usr/bin/darwin-amd64-opm:. --confirm

W0128 10:32:10.349443  176319 manifest.go:414] Chose linux/amd64 manifest from the list.
To include the manifest list digest: sha256:490035a0394c1f6aa32f193b61a28f983801ef467339729d4a724bb4b3ec7f7e, use --keep-manifest-list=true

$ ll
total 156072
-rw-r-----. 1 durant durant 67038328 Jan 15 19:30 darwin-amd64-opm
-rw-r-----. 1 durant durant 92769624 Jan 15 19:31 windows-amd64-opm
```